src/main/java/pvs/app/controller/GithubApiController.java

```diff
@@ -88,6 +88,44 @@ public GithubApiController(GithubApiService githubApiService, GithubCommitServic
         }
     }
 
+    @GetMapping("/github/commits/branchName/{repoOwner}/{repoName}")
+    public ResponseEntity<List<String>> getBranchesName(@PathVariable("repoOwner") String repoOwner, @PathVariable("repoName") String repoName) {
```

Comment:
Confusing about why this API needs `branchName`? In my opinion, the main action of this API is `get branch list from a specific repo of provided owner`, so the `branchName` will be one of the answer of its response which may not be involved in the request path or param.
So suggest remove this field.

```suggestion
    @GetMapping("/github/commits/{repoOwner}/{repoName}")
    public ResponseEntity<List<String>> getBrancheNames(@PathVariable("repoOwner") String repoOwner, @PathVariable("repoName") String repoName) {
```

Reply:
Does the url work if we removes `branchName` from @ GetMapping url and as same as the url of the function `getCommits`?